src/components/avatar/index.tsx

```diff
@@ -44,6 +46,35 @@ export enum BadgePosition {
 
 const DEFAULT_BADGE_SIZE = 'pimpleBig';
 
+export type NameToColorProps = {
+  /**
+   * Hash the name (or label) to get a color, so each name will have a specific color.
+   * Default is false.
+   */
+  useHashBackgroundColor: boolean;
```

Comment:
missing `?`
```suggestion
  useHashBackgroundColor?: boolean;
```

Also, I think the flag prop should not be part of the config but alongside it. 

```
<Avatar useHashBackgroundColor hashColorConfig={....} name={...} />
```

Reply:
Moved (renamed to `useAutoColorsConfig`)